src/styles/blocks/stars.css

```diff
@@ -0,0 +1,29 @@
+.stars {
+  display: flex;
+  position: center;
+}
+
+.stars__star {
```

Comment:
What is see that you background-images does reapeat. There is property to avoid that `background-repeat: no-repeat;`.